Makefile

```diff
@@ -0,0 +1,268 @@
+#
```

Comment:
This file is a standard SGX makefile with minimal  changes. Probably no need to review it in its original form.